app/src/ui/diff/text-diff.tsx

```diff
@@ -1498,17 +1501,15 @@ export class TextDiff extends React.Component<ITextDiffProps, ITextDiffState> {
         // If the text has changed the gutters will be recreated
         // regardless but if it hasn't then we'll need to update
         // the viewport.
-        if (this.props.diff.text === prevProps.diff.text) {
+        if (isSameDiff) {
           this.updateViewport()
         }
       }
     }
 
-    if (this.props.diff.text !== prevProps.diff.text) {
+    if (isSameDiff) {
```

Comment:
I think a `!` is missing here:
```suggestion
    if (!isSameDiff) {
```

Reply:
🤦